packages/noya-state/src/reducers/application.ts

```diff
@@ -134,6 +134,8 @@ export type Action =
       sharedStyleId: string,
       style: Sketch.Style | undefined,
     ]
+  | [type: 'duplicateColorSwatch', id: string[]]
```

Comment:
I think we mostly call them `Swatch` and not `ColorSwatch` in actions, e.g. `setSwatchName`, `removeSwatch`, `groupSwatches`. So let's call this `duplicateSwatch`.